core/src/main/kotlin/org/evomaster/core/EMConfig.kt

```diff
@@ -2017,7 +2017,7 @@ class EMConfig {
 
     @Cfg("User provided external service IP.")
     @Experimental
-    @Regex("^127\\.((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){2}([3-9]|[0-9][0-9]|[0-2][0-5][0-5]?)\$")
+    @Regex("^127\\.((0\\.){2}([3-9]|[1-9]\\d|[12]\\d\\d)|0\\.([1-9]|[1-9]\\d|[12]\\d\\d)\\.([1-9]?\\d|[12]\\d\\d)|([1-9]|[1-9]\\d|[12]\\d\\d)(\\.([1-9]?\\d|[12]\\d\\d)){2})\$")
```

Comment:
i don't understand this regex :( not saying it is wrong, just that it is not easy to read. should split in different strings (eg `const val`) with some explanation on each step